bitnami/redis/Chart.yaml

```diff
@@ -39,4 +39,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 21.0.2
\ No newline at end of file
+version: 21.1.0
```

Comment:
Should we bump patch version instead?

Reply:
depends whether this is a "new feature" or fixing an existing one that was "broken" (as in, it didn't exist)

I don't mind either way